src/main/java/teammates/ui/controller/CreateImageUploadUrlAction.java

```diff
@@ -0,0 +1,40 @@
+package teammates.ui.controller;
+
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.util.Const;
+import teammates.common.util.GoogleCloudStorageHelper;
+
+public class CreateImageUploadUrlAction extends Action {
```

Comment:
Consider adding class javadoc for the new classes added, or even better, the old classes that are significantly changed in this PR (like `AdminEmail*`). You can consider using this format:
```
Action: ...
```
for `*Action`, and
```
Page data for ...
```
for `*PageData`.